queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals where name LIKE '%mon%';
```

Comment:
[OPTIONAL]

-`%mon%` is a great way to query whether names wnd with `mon`. However, it'll also check whether the name begins with `mon`. To run a query for when names end with `mon`, the following query 👇  would be more specific 😀 

```
SELECT * FROM animals WHERE name LIKE '%mon'; 
```

Reply:
Thank you for the feedback.